pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1103,6 +1106,16 @@ func initPublications(zedagentCtx *zedagentContext) {
 	}
 	getconfigCtx.pubZedAgentStatus.ClearRestarted()
 
+	zedagentCtx.pubEdgeNodeClusterConfig, err = ps.NewPublication(pubsub.PublicationOptions{
+		AgentName:  agentName,
+		Persistent: true,
```

Comment:
Why does this need to be persistent? That makes it much more painful to add fields to the EdgeNodeClusterConfig in future EVE releases.